_maps/QuietStreet/QuietStreet.yaml

```diff
@@ -0,0 +1,174 @@
+---
+name:
+  en: Quiet Street
+desc:
+  en: "Quiet Street: A sleepy Christmas village where monkeys are on a neighborhood watch for bloons. The great hero Vrej hails from this town."
+ruleSet: Standard
+theme: Mario
+initialCash: 2500
+targetAmount: 25000
+baseSalary: 300
+salaryIncrement: 200
+maxDiceRoll: 7
+frbFile1: quietstreet
+background: quietstreet
+mapIcon: qs_btd6_0
+tourMode:
+  bankruptcyLimit: 1
+  opponent1: Stella
+  opponent2: Slime
+  opponent3: Kiryl
+  clearRank: 3
+music:
+  download: https://drive.google.com/u/0/uc?id=1yRtD5d5KuuGIhl7MwT1SHy9OzhRSaNtN&export=download
```

Comment:
the download url is not needed since the brstms are already available in the background.zip